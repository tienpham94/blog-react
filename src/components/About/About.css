.heading {
  font-size: 3rem;
}

.bio {
  font-size: 1.8rem;
  line-height: 1.7;
  color: #333;
}
