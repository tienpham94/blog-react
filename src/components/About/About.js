import React from 'react'

import styles from './About.css'

const About = () => (
  <div>
    <h1 className={styles.heading}>Hi, I am Kristian!</h1>
    <p className={styles.bio}>This section is all about me, yay!</p>
  </div>
)

export default About
