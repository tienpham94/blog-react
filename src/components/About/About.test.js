import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import About from './About'

test('About component should render as expected', () => {
  const component = shallow(<About />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
});
