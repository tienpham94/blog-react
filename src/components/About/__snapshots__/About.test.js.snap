exports[`test About component should render as expected 1`] = `
<div>
  <h1
    className="heading">
    Hi, I am Tien Pham
  </h1>
  <p
    className="bio">
    This section is all about me, yay!
  </p>
</div>
`;
